ibm/mas_devops/roles/ocp_setup_mas_deps/defaults/main.yml

```diff
@@ -2,6 +2,7 @@
 airgap_install: false
 
 cluster_type: "{{ lookup('env', 'CLUSTER_TYPE') }}"
+cluster_name: "{{ lookup('env', 'CLUSTER_NAME') }}"
```

Comment:
I don't see cluster_name being used anywhere in the changed files in this role, what's the reason for adding this variable?

Reply:
removed this unused var